modules/web/server/src/main/scala/scala/web/server/common/StaticRoutes.scala

```diff
@@ -17,7 +18,7 @@ import org.http4s.StaticFile
 import org.http4s.headers.`Cache-Control`
 import org.http4s.server.middleware.GZip
 
-class StaticRoutes[F[_]: Sync](
+class StaticRoutes[F[_]: Sync: Compression](
```

Comment:
What is this? `Compression`?

Reply:
Yes, this is needed for Gzip middleware. The constraint must now be declared explicitly so you know that it is using it.